pypdf/_utils.py

```diff
@@ -588,3 +588,8 @@ def replace(self, new_image: Any, **kwargs: Any) -> None:
         self.name = self.name[: self.name.rfind(".")] + extension
         self.data = byte_stream
         self.image = img
+
+
+def default_annotation_filter_function(page: Any, an: Any, obj: Any) -> bool:
```

Comment:
I do not see any advantage to have this function in _utils. It could be sufficient as a internal function next to _annotation_filter_function